### DISCUSSION TITLE
Popup

### DISCUSSION THREAD

**Discussion by Grumpypjs08** [Budibase/budibase#1344] [Ideas]
How can i make a popup.

**Comment by joebudi** [comment]
I am moving this to the Request feature category.

To answer your question, it is not possible to create a popup/model/dialogue box right now, but it is something we plan to implement in the future.

I am leaving this open until the functionality is released.

**Reply by Etkiny** [reply]
Hi, did you guys iplement popup? I could not find it in documantation. If not, do you still plan to add it?

**Reply by mjashanks** [reply]
We implemented the ability to show a screen in a  modal.

So:
1. Create a screen
2. From a button on another screen, click `Define Actions` and add a `Navigate To` action
3. Complete the URL and check the checkbox `Show in modal`

Inside your modal-screen, you can use a button action called `Close screen modal` when required.

**Reply by elapse-d** [reply]
Hello,

How do you create a temporary popup like a notification ?

**Reply by pbrink231** [reply]
@thomas10-10, I couldnt find a way.  I made an issue request for it here #7371.  I currently use the update row action without putting anything in the tables which still gives me a "Row Updated" alert without doing anything.
